prysm/nfs/validator-deployment.yaml

```diff
@@ -58,7 +58,7 @@ spec:
           secretName: wallet-password-1
           items:
           - key: password
-            path: password.txt
+            path: wallet_pass.txt
```

Comment:
can we change it to `wallet_password` or `wallet_pwd`?

Reply:
I changed both yamls and helm chart.